src/client/app/components/groups/CreateGroupModalComponent.tsx

```diff
@@ -103,6 +105,38 @@ export default function CreateGroupModalComponent(props: CreateGroupModalCompone
 	const [graphicUnitsState, setGraphicUnitsState] = useState(graphicUnitsStateDefaults);
 	/* End State */
 
+	// Sums the area of the group's deep meters. It will tell the admin if any meters are omitted from the calculation,
+	// or if any other errors are encountered.
+	const handleAutoCalculateArea = () => {
+		if(state.deepMeters != undefined && state.deepMeters.length > 0) {
+			if(state.areaUnit != AreaUnitType.none) {
+				let areaSum = 0;
+				let notifyMsg = '';
+				state.deepMeters.forEach(meterID => {
+					const meter = metersState[meterID];
+					if(meter.area > 0) {
+						if(meter.areaUnit != AreaUnitType.none) {
+							areaSum += meter.area * getAreaUnitConversion(meter.areaUnit, state.areaUnit);
+						} else {
+							notifyMsg += '\n' + meter.identifier + translate('group.area.calculate.error.unit');
```

Comment:
I believe this cannot happen with just meters added via the web interface, but pre-1.0 meters may have an area but no unit, and maybe imported meters as well.

In short I don't think it should happen but should still be checked anyways, since it will break the code here if a meter has area but no area unit.